packages/ws-server/src/controllers/index.ts

```diff
@@ -56,7 +56,12 @@ const handleSendingRequestsToRelay = async ({
 
   try {
     const resolvedParams = resolveParams(method, params);
-    const txRes = await relay.eth()[method.split('_')[1]](...resolvedParams, requestIdPrefix);
+    let txRes;
+    if (method.split('_')[0] === 'eth') {
```

Comment:
An if isn't sufficient here and could lead to future bugs.
The relay supports `eth_`, `debug_`, `get_` and `web3_` prefixes.

You don't have to yet support all of them but we should be more specific in our execution pathway management.

Reply:
Updated.